### PR TITLE
v1 of the JSON Schema Ambassadors Program

### PR DESCRIPTION
**Pull request by benjagm**
**GitHub Issue:** # / NA

**Summary**: v1 of the JSON Schema Ambassadors Program

**Do you think resolving this issue might require an [Architectural Decision Record (ADR)](https://github.com/json-schema-org/community/blob/main/CONTRIBUTING.md#key-architectural-decisions)? (significant or noteworthy)**

No

### CODE REVIEW

File: programs/ambassadors/ambassadors-program.md
Comment (Julian) [inline programs/ambassadors/ambassadors-program.md:7]:
```suggestion
- Promote collaboration among the JSON Schema user base.
```

File: programs/ambassadors/ambassadors-program.md
Comment (Julian) [inline programs/ambassadors/ambassadors-program.md:9]:
```suggestion
- Speak at community events or write technical content (e.g. blog posts) that solves community needs.
```

File: programs/ambassadors/ambassadors-program.md
Comment (Julian) [inline programs/ambassadors/ambassadors-program.md:27]:
```suggestion
- Articles, videos, and podcasts can be published on [JSON-Schema.org/blog](https://json-schema.org/blog) or other domains. 
```

File: programs/ambassadors/ambassadors-program.md
Comment (Julian) [inline programs/ambassadors/ambassadors-program.md:28]:
```suggestion
- Talks and presentations can be displayed at different events or workshops. Within them, JSON Schema should appear at the core of the presentation. The presentation should not be being used as a disguised marketing tool for other products/projects.
```

File: programs/ambassadors/ambassadors-program.md
Comment (Julian) [inline programs/ambassadors/ambassadors-program.md:30]:
```suggestion
- If we do not notice or accidentally miscount a contribution, just let us know.
```

File: programs/ambassadors/ambassadors-program.md
Comment (Julian) [inline programs/ambassadors/ambassadors-program.md:34]:
```suggestion
1. If contributors publish 3 articles and drive 1 initiative about JSON Schema in eight months, they become ambassadors for the whole year.
```

File: programs/ambassadors/ambassadors-program.md
Comment (Julian) [inline programs/ambassadors/ambassadors-program.md:35]:
```suggestion
2. If contributors make 1 presentation, write 2 articles, and drive 1 initiative, they become ambassadors for the whole year.
```

File: programs/ambassadors/ambassadors-program.md
Comment (Julian) [inline programs/ambassadors/ambassadors-program.md:36]:
```suggestion
3. If you were accepted as ambassador on the 5th of May 2023, you have already fulfilled the requirements. On the 5th of May 2024, we will check if you qualify again after the 5th of May 2023.
```

File: programs/ambassadors/ambassadors-program.md
Comment (Julian) [inline programs/ambassadors/ambassadors-program.md:41]: Have we considered hosting this on the website so it has a nicer URL?

File: programs/ambassadors/ambassadors-program.md
Comment (Julian) [inline programs/ambassadors/ambassadors-program.md:46]:
```suggestion
- Invitation to [the JSON Schema organization](https://github.com/orgs/json-schema-org/people).
```

File: programs/ambassadors/ambassadors-program.md
Comment (Julian) [inline programs/ambassadors/ambassadors-program.md:48]:
```suggestion
- Receive additional swag for conference or workshop participants (Swag availability date to be confirmed).
```

File: programs/ambassadors/ambassadors-program.md
Comment (Julian) [inline programs/ambassadors/ambassadors-program.md:50]:
```suggestion
- All our respect for your contributions!
```

**Review by Julian — COMMENTED** [review]
Cool! Looks nice. Left minor comments. I think the answer will be "of course" but it'll be nice to see if there's some immediate interest in the program (i.e. some people who are itching to be one!) 

But definitely exciting to see this!

File: programs/ambassadors/ambassadors-program.md
Comment (benjagm) [reply programs/ambassadors/ambassadors-program.md:41]: Great idea! I just filled a new issue for this enhancement. https://github.com/json-schema-org/website/issues/278

File: programs/ambassadors/ambassadors-program.md
Comment (Relequestual) [inline programs/ambassadors/ambassadors-program.md:15]: While this may be accurate for AsyncAPI, this isn't right for JSON Schema, given that (process of adding someone to the TSC) isn't active right now.

File: programs/ambassadors/ambassadors-program.md
Comment (Relequestual) [inline programs/ambassadors/ambassadors-program.md:16]: "community repositories"?

File: programs/ambassadors/ambassadors-program.md
Comment (Relequestual) [inline programs/ambassadors/ambassadors-program.md:54]: What if we say the TSC AND existing Ambassadors?

**Review by Relequestual — COMMENTED** [review]
This looks like a great start. I have left a few comments.

1. I think we need to specify that someone who is a candidate and is approved/accepted still has to accept for them to become an Ambassador. We aren't just forcing it on people who write nice things about us =]
2. Does this differ from a [Champion](https://github.com/orgs/json-schema-org/projects/12/views/3?pane=issue&itemId=17857484)? Or are these the same thing? (I think there is a benefit to having both with different requirements and expectations)

File: programs/ambassadors/ambassadors-program.md
Comment (benjagm) [reply programs/ambassadors/ambassadors-program.md:15]: Thanks for the heads up. I'll change it.

File: programs/ambassadors/ambassadors-program.md
Comment (benjagm) [reply programs/ambassadors/ambassadors-program.md:54]: Sounds great

**Comment by benjagm** [comment]
> 1. I think we need to specify that someone who is a candidate and is approved/accepted

I think is that person who make the PR, and that way the person is asking to join.

> 2\. Does this differ from a [Champion](https://github.com/orgs/json-schema-org/projects/12/views/3?pane=issue&itemId=17857484)?

We can achieve the same with one single program. We are a small project to launch 2 programs like these, when they have a lot of commonalities. It s better to add more categories of types of ambassadors in my opinion. What are in your opinion the differences between the 2 programs?

**Review by Julian — APPROVED** [review]

**Review by Relequestual — APPROVED** [review]
Excited to see this take shape! Thanks for your work on this =D

**Comment by Relequestual** [comment]
> > 1. I think we need to specify that someone who is a candidate and is approved/accepted
> 
> I think is that person who make the PR, and that way the person is asking to join.

But, we may have nominations. I guess we could ask people to submit a PR in that case.
I think this is fine. It doesn't need to be more complicated =]

> 
> > 2. Does this differ from a [Champion](https://github.com/orgs/json-schema-org/projects/12/views/3?pane=issue&itemId=17857484)?
> 
> We can achieve the same with one single program. We are a small project to launch 2 programs like these, when they have a lot of commonalities. It s better to add more categories of types of ambassadors in my opinion. What are in your opinion the differences between the 2 programs?

I see Champions as having a less formal process, no swag, one time formal recognition by the TSC or active community. "[NAME] is recognised as JSON Schema Community Champion 2023" - No review, just one off.

File: programs/ambassadors/ambassadors-program.md
Comment (gregsdennis) [inline programs/ambassadors/ambassadors-program.md:45]: This is a bit concerning.  As far as I'm aware, members have GitHub write access like committing directly to repo branches (without making a fork), merging PRs, etc.  I'm not sure we want to extend that authority to people outside the core team / TSC.

**Review by gregsdennis — COMMENTED** [review]
Mostly great.  Just one concern.

File: programs/ambassadors/ambassadors-program.md
Comment (benjagm) [reply programs/ambassadors/ambassadors-program.md:45]: > This is a bit concerning.

I hear you and you are right. I'll replace this point by Visibility in the JSON Schema Ambassadors page of the JSON Schema website.

Comment (Relequestual) [reply programs/ambassadors/ambassadors-program.md]: It doesn't automatically. We can manage it on a team basis. We can have a team which has read only access to the public repos. It may give them permissions to crates issue labels for example, but I think even that can be limited.

Personally, I think if we can do it without giving permissions, we should. I've always liked having logos on my GitHub profile. There's a sense of pride in association.

Comment (gregsdennis) [reply programs/ambassadors/ambassadors-program.md]: My concern is authority. If that can be managed, I'm okay with this.

**Comment by Relequestual** [comment]
I don't think any discussions about a champions program should hold up launching the ambassadors progam though! =D

**Comment by benjagm** [comment]
> Microsoft MVP, Google Developer Experts, GitHub Stars. You can't apply for these.
> I think it would be very minimal. Small number of slots, suggestions from TSC and ambassadors only. Maybe swag would make senes. Option to claim some monetary award.

I think you are mixing terms with those programs. Actually become a Microsoft MVP or Google Developer Expert is not easy. There is a formal process and they need to specific contributions to complete that year.

In addition, those program you mention, they don't have an Ambassador program on top ... Ambassador, Champions... they are all the same. I think launching 2 programs Ambassadors and champions at the same time may be confusing.

**Comment by Relequestual** [comment]
After a short discussion, I agree it makes sense to only have one program, which may at some point have different parts.

File: programs/ambassadors/ambassadors-program.md
Comment (Relequestual) [reply programs/ambassadors/ambassadors-program.md]: By default, any member or team member, provides no additional permissions, apart from creating a repo.
We could remove the permission for all members to create repos, which I think might actually be the right thing to do anyway. We don't need to create new repos often.

**Comment by benjagm** [comment]
> But, we may have nominations. I guess we could ask people to submit a PR in that case.
> I think this is fine. It doesn't need to be more complicated =]

I just pushed a new version clarifying this aspect. I was thinking in rename it to champions but finally I leave it as ambassadors because I think there is no real difference in the final outcome.

**Review by Relequestual — APPROVED** [review]
This is shaping up very nicely. Thanks for your work on this!

**Comment by Relequestual** [comment]
Associated budget discussion/TSC vote: https://github.com/json-schema-org/community/issues/678

**Pull request merged**
